### DISCUSSION TITLE
Idea about cryptography

### DISCUSSION THREAD

**Discussion by martinweihrauch** [miroslavpejic85/p2p#32] [General]
Have you considered also encrypting the image stream?
BTW, why did you encrypt the text string "yourself"?
In C#, there is the CryptoStream class, which could handle this for you.

Thx
Martin

**Comment by miroslavpejic85** [comment]
Hi @martinweihrauch,

I made this project about 10 years ago, then I left it in a drawer, and I decided to make it open-source about a year ago. 

Surely it can be improved. I'm also a little rusty in C# as I currently work on other programming languages. 

Feel free to send me any improvement PR! :) 

Thank you.
